core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -571,13 +570,17 @@ private synchronized void addFailedMutations(MutationSet failedMutations) {
     }
   }
 
-  private class FailedMutations extends TimerTask {
+  private class FailedMutations {
 
     private MutationSet recentFailures = null;
     private long initTime;
+    private final Runnable task;
 
     FailedMutations() {
-      jtimer.schedule(this, 0, 500);
+      task = Threads.createNamedRunnable("failed mutationBatchWriterLatencyTimers handler", () -> {
+        run();
+      });
```

Comment:
Would this work?

```suggestion
      task = Threads.createNamedRunnable("failed mutationBatchWriterLatencyTimers handler", this::run);
```

Reply:
That should work, I can make that change.